src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -200,6 +202,7 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
 
   // Feature flags
   const complianceEnabled = useFlag('image-builder.compliance.enabled');
+  // const isAAPRegistrationEnabled = useFlag('image-builder.aap.enabled') || true; // Default to true for now, remove when flag is implemented
```

Comment:
Just checked and the flag is currently disabled in both prod and stage unleash, how should we set it up? Enabled for stage preview only?

Reply:
We'll also need to set the flag for tests, leaving this here so we don't forget to take a look at that :sweat_smile:

Reply:
The flags are set to enable AAP in stage-preview only for now :+1: